downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -87,6 +87,27 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 
 ### Improvements
 
+#### CLI
+##### Refactor bal format command
+Introduced new flags to `bal format` command and updated the format of the command. New `bal format` command would look like below,
+
+`bal format [OPTIONS] [<package>|<module>|<source-file>]`
```

Comment:
```suggestion
`bal format [OPTIONS] [<package>|<module>|<source-file>]`
```
How the backticks are added is incorrect. Please check in other Learn pages. We have to have 3 backticks at the top and bottom to add a code snippet.